src/MiniJuvix/Syntax/MiniC/Language.hs

```diff
@@ -221,6 +267,15 @@ typeDefType typName declName =
       _declInitializer = Nothing
     }
 
+typeDefType' :: Text -> Declaration
```

Comment:
I didn't find any usage of this function. In case, you need it, is it better if you reuse the one above, i.e.,

```haskell
typeDefType' :: Text -> Declaration 
typeDefType' typName = (typeDefType typName "") {_declName = Nothing}
```

Reply:
Thanks - this can be removed.